app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+
+  def self.ingredient_search(search_string)
+    data = FoodService.foods_by_ingredient(search_string)
+
+    food_results = data[:foods]
+
+    @foods = food_results.map do |food|
```

Comment:
Because this is a facade with class methods, you don't need to name these with `@` as they aren't instance variables that need to be accessible elsewhere